x-pack/legacy/plugins/siem/public/pages/hosts/details/body.tsx

```diff
@@ -22,9 +22,7 @@ const HostDetailsBodyComponent = React.memo<HostDetailsBodyComponentProps>(
     filterQueryExpression,
     from,
     isInitializing,
-    match: {
-      params: { detailName, tabName },
-    },
+    detailName,
```

Comment:
Note: In these cases we have to remove match and can longer pass it down. Instead we have to destructure the match into its constituent pieces of things such as strings so React.memo does not do additional rendering.